torchbox/wagtail-torchbox
# Sub-proposition page - Process block tidy-up

[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1299668523)

### Description of Changes Made

This MR removes the two fields: `process_block_image` & `processes_section_embed_url` from the `ProcessesBlock`.

### How to Test

- When you add a process block to a sub proposition page, the *Use process block image* and *embed URL* options should not be available.
- On the frontend, all other elements should render as before, without any references to a process block image or embed.

### Screenshots

<details>
  <summary>Before</summary>

![image](https://github.com/torchbox/wagtail-torchbox/assets/7713776/a45cbb4c-f709-463a-8af8-7d4a6e94bf01)

</details>

<details>
  <summary>After</summary>

![Screenshot from 2023-11-02 16-18-57](https://github.com/torchbox/wagtail-torchbox/assets/7713776/6b13f4ab-97f2-4e78-b0f3-cff6e2b73418)


</details>

### MR Checklist
